layouts/shortcodes/issues-security.html

```diff
@@ -0,0 +1,19 @@
+<table class="security-cves">
+  <caption style="text-align:center">Table: Official Kubernetes CVE List</caption>
+  <thead>
+  <tr>
+    <th>{{ T "cve_id" }}</th>
+    <th>{{ T "cve_issue_summary"}}</th>
+    <th>{{ T "cve_issue_details" }}</th>
+  </tr>
+  </thead>
+  <tbody>
+  {{ range $issues := getJSON "static/security/official-cve-feed.json" }}
+  <tr>
+    <td><a href="{{ .cve_url }}">{{ .cve_id | htmlEscape }}</a></td>
+    <td>{{ .summary | htmlEscape }}</td>
```

Comment:
(nit) Mark this text as English - it's not localized.
```suggestion
    <td lang="en">{{ .summary | htmlEscape }}</td>
```